src/test/java/com/wikia/webdriver/common/core/api/EditToken.java

```diff
@@ -42,9 +42,9 @@ public EditToken(String username) {
   private static ResponseHandler<String> extractEditToken() {
     return res -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this lambda expression number of lines from 21 to at most 20. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)